src/jdk.httpserver/share/classes/module-info.java

```diff
@@ -52,7 +52,8 @@
  *    Usage: java -m jdk.httpserver [-b bind address] [-p port] [-d directory]
  *                                  [-o none|info|verbose] [-h to show options]
  *    Options:
- *    -b, --bind-address    - Address to bind to. Default: 0.0.0.0 (all interfaces).
+ *    -b, --bind-address    - Address to bind to. Default: 127.0.0.1 (loopback).
```

Comment:
This assumes that the machine on which the server is run has IPv4 configured. It might not be the case. It can also depend on whether -Djdk.net.preferIPv6Addresses=true is specified on the java command line. Maybe this should be: `127.0.0.1 (or ::1), (loopback).`